test/libponyc/verify.cc

```diff
@@ -492,3 +492,11 @@ TEST_F(VerifyTest, FFICallPartialWithNonPartialDeclaration)
 
   TEST_ERRORS_1(src, "call is partial but the declaration is not");
 }
+
+TEST_F(VerifyTest, LambdaTypeGenericAliased)
+{
+  const char *good =
```

Comment:
To be consistent with the other tests, could you use `src` as the name of the variable that holds the string for this source code, rather than `good`?